tests/Console/ImportLegacyCampaignsCommandTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Tests\Console;
+
+use Carbon\Carbon;
+use Tests\TestCase;
+use Rogue\Models\Campaign;
+
+class ImportLegacyCampaignsCommandTest extends TestCase
+{
+    public function testImportingLegacyCampaigns()
+    {
+        // Run the legacy campaign import command
+        $this->artisan('rogue:ashescampaignsimport', ['path' => 'tests/Console/example-legacy-campaigns.csv']);
```

Comment:
yesssss thank you for making tests for this!! 🙌

Reply:
😄